packages/components/src/Notification/GlobalNotification/_docs/GlobalNotification.stickersheet.stories.tsx

```diff
@@ -0,0 +1,119 @@
+import React from "react"
+import { Meta } from "@storybook/react"
+import {
+  StickerSheet,
+  StickerSheetStory,
+} from "~storybook/components/StickerSheet"
+import {
+  GlobalNotificationProps,
+  GlobalNotification,
+} from "../GlobalNotification"
+
+export default {
+  title: "KAIO-staging/Notifications/Global Notification",
+  parameters: {
+    chromatic: { disable: false },
+    controls: { disable: true },
+    a11y: {
+      config: {
+        rules: [
+          {
+            // Looks like axe is having issues with the overlapping elements in stickersheets causing false positives.
+            id: "color-contrast",
+            enabled: false,
+          },
+        ],
+      },
+      // Fade-in animation has colour contrast issues.
+      timeout: 1000,
+    },
```

Comment:
Found a fix for this if you want to try apply it: https://github.com/cultureamp/kaizen-design-system/pull/4011/files#diff-1cd3369f4eb2d832997e6938c6672850460051ff84c4bca996a85327f27f3a0cR40